website/package.json

```
@@ -27,7 +27,7 @@
     "language:lint": "cd ../ && alex && case-police 'docs/*.md' -p brands,general,products,softwares",
     "language:lint:versioned": "cd ../ && alex . && case-police '**/*.md' -p brands,general,products,softwares",
     "ci:lint": "yarn lint && yarn lint:versioned && yarn language:lint:versioned",
-    "pwa:generate": "npx pwa-asset-generator ./static/img/header_logo.svg ./static/img/pwa --background #20232a --icon-only -opaque true --maskable true --type png"
+    "pwa:generate": "npx pwa-asset-generator ./static/img/header_logo.svg ./static/img/pwa --padding 12.5% --background #20232a --icon-only -opaque true --maskable true --type png"
```

```suggestion
    "pwa:generate": "npx pwa-asset-generator ./static/img/header_logo.svg ./static/img/pwa --padding '12.5%' --background #20232a --icon-only -opaque true --maskable true --type png"
```

From the command help, it looks like the padding parameter value need to wrapped with quotes. 

This might be the issue which cause malformed command output.